site/src/docs/components/logo/index.mdx

```diff
@@ -14,38 +14,37 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 ### Example
 
 <PlaygroundPreview>
-  <Logo language="fi" size="medium" />
+  <Logo src={logoFi} size="medium" alt="logo"/>
 </PlaygroundPreview>
 
-The logo is used to identify the site or application as an official City of Helsinki service.
+The logo is used to identify the site or application.
 See the <InternalLink size="M" href="/foundation/visual-assets/logo">logo visual asset documentation</InternalLink> for more information about principles of using the City of Helsinki logo.
 
 ### Principles
 
-- Logo is automatically included in HDS Navigation and HDS Footer (not yet released) components. **Use the Logo component sparingly in other parts of the service.**
+- Logo should be used in HDS Navigation and HDS Footer (not yet released) components. **Use the Logo component sparingly in other parts of the service.**
 - **You should include the service name next to the logo.** This helps the user more easily identify which city service they are currently using.
-- HDS Logo component includes translations for both Finnish and Swedish. **Use translation depending on the locale the user has chosen.** Swedish and Scandinavian languages use the `sv` variant while other languages use `fi` variant.
+- The Logo component does not include translations itself but localization can be achieved by changing the logo src accordingly.
```

Comment:
Should we mention here that users can use their own logos if they wish with the `src` prop?